src/Detector/Ruby/RubyPlatformDetectorResult.cs

```diff
@@ -19,5 +19,10 @@ public class RubyPlatformDetectorResult : PlatformDetectorResult
         /// Gets or sets the value if a bundler version specified in 'Gemfile.lock' in the repo.
         /// </summary>
         public string BundlerVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets the value if a bundler version specified in 'Gemfile.lock' in the repo.
```

Comment:
Is this correct?

Reply:
It's not, thanks for catching it, will update